packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -2811,6 +2820,57 @@ protocol version:      ${this._protocolVersion}`;
 		}
 	}
 
+	/** Handles the receipt of a MultilevelCC Set or Report */
+	private handleMultilevelSwitchCommand(command: MultilevelSwitchCC): void {
+		// Retrieve the endpoint the command is coming from
+		const sourceEndpoint =
+			this.getEndpoint(command.endpointIndex ?? 0) ?? this;
+		if (command instanceof MultilevelSwitchCCSet) {
+			this.driver.controllerLog.logNode(this.id, {
+				endpoint: command.endpointIndex,
+				message: "treating MultiLevelSwitchCCSet::Set as a value event",
+			});
+			this._valueDB.setValue(
+				getMultilevelSwitchCCCompatEventValueId(command.endpointIndex),
+				command.targetValue,
+				{
+					stateful: false,
+				},
+			);
+			return;
+		}
+		if (command instanceof MultilevelSwitchCCStartLevelChange) {
+			this.driver.controllerLog.logNode(this.id, {
+				endpoint: command.endpointIndex,
+				message:
+					"treating MultilevelSwitchCCStartLevelChange::Set as a value event",
+			});
+			this._valueDB.setValue(
+				getMultilevelSwitchCCCompatEventValueId(command.endpointIndex),
+				command.direction,
+				{
+					stateful: false,
+				},
+			);
+			return;
+		}
+		if (command instanceof MultilevelSwitchCCStopLevelChange) {
+			this.driver.controllerLog.logNode(this.id, {
+				endpoint: command.endpointIndex,
+				message:
+					"treating MultilevelSwitchCCStopLevelChange::Set as a value event",
+			});
+			this._valueDB.setValue(
+				getMultilevelSwitchCCCompatEventValueId(command.endpointIndex),
+				"stop",
+				{
+					stateful: false,
+				},
+			);
+			return;
+		}
```

Comment:
Let's not water down the meaning of the event value too much, it should be limited to the value received via a Set command.
I'd prefer having start/stopLevelChange as a new `notification` type, like for Powerlevel CC and Entry Control CC - see point 3. in my comment in the issue.